Vagrantfile

```diff
@@ -45,5 +51,15 @@ EOF
         && make install-criu \
         && cd - \
         && rm -rf /usr/src/criu
+
+    # Delegate cgroup v2 controllers to rootless user via --systemd-cgroup
+    mkdir -p /etc/systemd/system/user@.service.d/delegate.conf
+    cat > /etc/systemd/system/user@.service.d/delegate.conf << EOF
+[Service]
+# default: Delegate=pids memory
+# NOTE: delegation of cpuset requires systemd >= 244 (Fedora >= 32, Ubuntu >= 20.04). cpuset is ignored on Fedora 31.
```

Comment:
I tried to bump up Vagrantfile to use Fedora 32, but could not compile criu on Fedora 32.
Let me try that in yet another PR.

Reply:
https://github.com/opencontainers/runc/pull/2363 is merged now